reporter-web-app/config/webpack.config.js

```diff
@@ -654,7 +654,7 @@ module.exports = function (webpackEnv) {
 
             const result = data.replace(
               /(<script.*? id="ort-report-data">)([\s\S]*?)(<\/script>)/g,
-              '$1ORT_REPORT_DATA_PLACEHOLDER$3'
+              '<script type="application/gzip" id="ort-report-data">ORT_REPORT_DATA_PLACEHOLDER$3'
```

Comment:
Has the `$1` been dropped here intentionally?

Reply:
$1 still holds "application/json" as for WebApp development we use plain JSON so we can easily debug issues. this code generates the WebApp template which is used by Kotlin code which will inject "application/gzip".

Reply:
Ah, sorry, I didn't look closely enough at the replacement logic before. Then it would make more sense to me to change "application/json" to "application/gzip" in the template, and keep this replacement string as it was before.

Reply:
I agree, it should be replaced in the template.

Reply:
You mean you want to do the "application/json" to "application/gzip" replacement in the Koltin code?